py_zipkin/zipkin.py

```diff
@@ -292,6 +323,18 @@ def update_binary_annotations_for_root_span(self, extra_annotations):
         self.logging_context.binary_annotations_dict.update(extra_annotations)
 
 
+class zipkin_client_span(zipkin_span):
+    def __init__(self, *args, **kwargs):
+        kwargs['include'] = ('client',)
```

Comment:
Could you check for 'include' in kwargs and error if it's there? Does that sound like a good idea?